verifier/src/lib.rs

```diff
@@ -93,11 +93,11 @@ pub enum VerificationError {
         /// The actual measurement that was present
         actual: MrEnclave,
     },
-    /// The MRSIGNER measurement did not match expected:{expected} actual:{actual}
-    MrSignerMismatch {
-        /// The expected measurement
+    /// The MRSIGNER key did not match expected:{expected} actual:{actual}
+    MrSignerKeyMismatch {
```

Comment:
I'm up for ideas on the name of just the key hash.
I also considered "measurement" since that what the C API type is, but measurement felt ambiguous with key hash versus (key hash + product id + svn)